test/foundry/dapp/Application.t.sol

```diff
@@ -497,6 +529,207 @@ contract ApplicationTest is ERC165Test {
         _appContract.executeOutput(output, proof);
     }
 
+    function testEtherTransferToENS() external _resolveENS {
```

Comment:
This function could reuse a good chunk from the `testExecuteEtherTransferVoucher` function. Maybe move the code block from there to an internal function, and call it from here as well?